pages/pricing-beta/_plans.html

```diff
@@ -0,0 +1,194 @@
+<div class="row row-page-header">
```

Comment:
Same here. This can be broken into several files that are `include`-ed to make things more readable.